sygnal/gcmpushkin.py

```diff
@@ -707,6 +707,8 @@ def _build_data(
                 for attr, value in data["content"].items():
                     if not isinstance(value, str):
                         continue
+                    if len(value) > MAX_BYTES_PER_FIELD:
+                        value = value[0:MAX_BYTES_PER_FIELD]
```

Comment:
This will do for now, but we should a) handle this in unicode, and b) and an ellipsis at the end.

Reply:
Ah - I had just borrowed this from the other places we already do truncation. 
I'll update this to add an ellipsis, but leave the unicode for a future PR.